ext/js/pages/settings/anki-controller.js

```diff
@@ -505,6 +505,22 @@ export class AnkiController {
                 return null;
         }
     }
+
+    /**
+     * @param {import('anki').Note[]} notes
+     * @returns {Promise<?((number | null)[] | null)>}
+     */
+    async addNotes(notes) {
```

Comment:
Would the function `addNote` above be able to call this function?

Reply:
Not sure what you mean by this. If you're asking why it uses `anki-controller.js` as a passthrough instead of calling `anki-connect.js` directly it's because `anki-controller.js` already has the instance information set up and it makes much more sense to just use that rather than duplicate its code.

Reply:
It's kind of unrelated to this PR or out of scope. The [`addNote` function](https://github.com/themoeway/yomitan/pull/895/files/894dfc8f396cd6fd83e92dbede80c9653397ea94#diff-10ae4205e4876099259ef91e9e10d7117a50d940da1d7b6d41e5a9f53ac3182aR122) above is defined as
```js
    /**
     * @param {import('anki').Note} note
     * @returns {Promise<?import('anki').NoteId>}
     */
    async addNote(note) {
        if (!this._enabled) { return null; }
        await this._checkVersion();
        const result = await this._invoke('addNote', {note});
        if (result !== null && typeof result !== 'number') {
            throw this._createUnexpectedResultError('number|null', result);
        }
        return result;
    }
```
Could change it to be 
```js
    /**
     * @param {import('anki').Note} note
     * @returns {Promise<?import('anki').NoteId>}
     */
    async addNote(note) {
        return addNotes([note]);
    }
```

![image](https://github.com/themoeway/yomitan/assets/2382537/f3d65d25-cc8a-4cc4-b741-5cdb3b6ab284)

Reply:
I'm almost certain this would break ankiconnect android.

Reply:
oh sorry i'm dumb i meant to comment on the `addNotes` function in `anki-connect.js` not `anki-controller.js`